### DISCUSSION TITLE
Can I dont change `savePath` when click save screenshot to a file (ctrl+s)

### DISCUSSION THREAD

**Discussion by shirobachi** [flameshot-org/flameshot#3040] [Q&A] [answered]
I want to if I click save ss to a file to this save only once ss in the location I choose. And next time use location from config file.

Why? 
I have checked in option saveAfterCopy so all screenshots are save in Downloads, but also sometimes I need to put one time screenshot in different location. I can make myself to just move last ss, but maybe I can change flameshot behawiourt?

this is my config:
```
[General]
contrastOpacity=188
copyPathAfterSave=true
disabledTrayIcon=true
drawColor=#ff0000
drawFontSize=14
drawThickness=5
ignoreUpdateToVersion=11.0.0
saveAfterCopy=true
savePath=/home/simon/Documents/
showDesktopNotification=false
showHelp=false
showStartupLaunchMessage=false
undoLimit=100
uploadHistoryMax=25
uploadWithoutConfirmation=true
useJpgForClipboard=false

[Shortcuts]
TYPE_COPY=Return
TYPE_PIN=`
```

Thanks and happpy new year!

**Comment by mmahmoudian** [comment, marked as answer]
you can use the `--path` argument in the commandline to get what you need. For example:

```sh
flameshot gui --path "/home/simon/Desktop/"
```

**Comment by shirobachi** [comment]
Thanks so much, are you aware if I can make enter confirm? I have it when run without `--path ..` but when run with I need use mouse and click confirm icon

Thanks again!

**Reply by mmahmoudian** [reply]
It is already set to `Enter`, but you can change it whatever you want:

![image](https://user-images.githubusercontent.com/390889/212544899-b9d20592-23be-4f34-ac9a-3007420fcf90.png)
